webpack.config.js

```diff
@@ -1,22 +1,76 @@
+const fs = require('fs')
 const path = require('path')
-const srcPath = path.join(__dirname, 'src') + path.sep
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const marked = require('meta-marked')
 const widgetWebpack = require('materia-widget-development-kit/webpack-widget')
-const entries = widgetWebpack.getDefaultEntries()
+const rules = widgetWebpack.getDefaultRules()
+const copy = widgetWebpack.getDefaultCopyList()
+
+const srcPath = path.join(__dirname, 'src') + path.sep
+const outputPath = path.join(process.cwd(), 'build')
 
-entries['player.js'] = [
-	srcPath+'player.coffee',
-	srcPath+'draw.coffee'
-]
+const customCopy = copy.concat([
+	{
+		from: path.join(__dirname, 'src', '_guides', 'assets'),
+		to: path.join(outputPath, 'guides', 'assets'),
+		toType: 'dir'
+	},
+])
 
-entries['creator.js'] = [
-	srcPath+'creator.coffee',
-	srcPath+'draw.coffee',
-	srcPath+'spectrum.custom.js'
-]
+const entries = widgetWebpack.getDefaultEntries()
+
+const entries = {
+	'creator.js': [
+		path.join(__dirname, 'src', 'spectrum.custom.js'),
+		path.join(__dirname, 'src', 'draw.coffee'),
+		path.join(__dirname, 'src', 'creator.coffee')
+	],
+	'player.js': [
+		path.join(__dirname, 'src', 'draw.coffee'),
+		path.join(__dirname, 'src', 'player.coffee')
+	],
+	'creator.css': [
+		path.join(__dirname, 'src', 'creator.html'),
+		path.join(__dirname, 'src', 'creator.scss')
+	],
+	'player.css': [
+		path.join(__dirname, 'src', 'player.html'),
+		path.join(__dirname, 'src', 'player.scss')
+	],
+	'guides/guideStyles.css': [
+		path.join(__dirname, 'src', '_guides', 'guideStyles.scss'),
+	],
+	'guides/player.temp.html': [
+		path.join(__dirname, 'src', '_guides', 'player.md')
+	],
+	'guides/creator.temp.html': [
+		path.join(__dirname, 'src', '_guides', 'creator.md')
+	]
+}
 
-// options for the build
-let options = {
+const options = {
+	copyList: customCopy,
 	entries: entries
 }
 
-module.exports = widgetWebpack.getLegacyWidgetBuildConfig(options)
+const generateHelperPlugin = name => {
+	const file = fs.readFileSync(path.join(__dirname, 'src', '_guides', name+'.md'), 'utf8')
+	const content = marked(file)
+
+	return new HtmlWebpackPlugin({
+		template: path.join(__dirname, 'src', '_guides', 'helperTemplate'),
+		filename: path.join(outputPath, 'guides', name+'.html'),
+		title: name.charAt(0).toUpperCase() + name.slice(1),
+		chunks: ['guides'],
+		content: content.html
+	})
+}
+
+let buildConfig = widgetWebpack.getLegacyWidgetBuildConfig(options)
+
+buildConfig.plugins.unshift(generateHelperPlugin('creator'))
+buildConfig.plugins.unshift(generateHelperPlugin('player'))
```

Comment:
These two lines were part of the old configuration, where we had to define a new plugin (above) and add it to the `buildConfig.plugins` array manually. The core MWDK widget-webpack handles that now, so these lines aren't required. It's sufficient to simply call:
```
let buildConfig = widgetWebpack.getLegacyWidgetBuildConfig(options)
module.exports = buildConfig
```

With `options` including the `copyList` and `entries` objects.